src/Tizen.NUI/src/public/Common/Layer.cs

```diff
@@ -341,6 +341,14 @@ public override void Remove(View child)
             {
                 throw new ArgumentNullException(nameof(child));
             }
+            if (child.GetParent() == null) // Early out if child parent is null.
+                return;
+
+            if (child.GetParent() != this)
+            {
+                //throw new System.InvalidOperationException("You have deleted a view that is not a child of this view.");
+                Tizen.Log.Error("NUI", "You have deleted a view that is not a child of this view.");
```

Comment:
please change this => `not a child of this view` to `not a child of this layer`

Reply:
Thanks for detail review.